docs/upgrade.adoc

```diff
@@ -14,6 +14,12 @@ for detailed instructions to perform this upgrade.
 Users who operate their own Cassandra cluster instead of using Cassandra distributed together with JanusGraph are not affected by this upgrade.
 This also does not change the different versions of Cassandra that are supported by JanusGraph (see <<version-compat>> for a detailed list of the supported versions).
 
+==== BerkeleyDB : Upgrade from 7.4 to 7.5
+The BerkeleyDB version has been updated, and it contains changes to the file format stored on disk.
+This file format change is forward compatible with previous versions of BerkeleyDB, so existing graph data stored with JanusGraph can be read in.
+However, once the data has been read in with the newer version of BerkeleyDB, those files can no longer be read by the older version.
```

Comment:
It might be nice to link the [change log](https://docs.oracle.com/cd/E17277_02/html/changelog.html) that was in the issue here

Reply:
OK, I added a link to the change log. I initially didn't do that as it only contains a single entry that is relevant here which is the upgrade from 7.4 to 7.5 and that only repeats what I already wrote here: The file format changed which is forward compatible but not backward compatible.
But I guess it doesn't hurt to still include it for completeness.